release-notes/6.0/cve.md

```diff
@@ -0,0 +1,29 @@
+# .NET 6 CVEs
+
+The .NET Team releases [monthly updates for .NET 6](https://github.com/dotnet/announcements/labels/.NET%206.0) on [Patch Tuesday](https://en.wikipedia.org/wiki/Patch_Tuesday). These updates often include security fixes. If you are on an older version, your app may be vulnerable.
```

Comment:
"The .NET Team releases [servicing updates for .NET 6](https://github.com/dotnet/announcements/labels/.NET%206.0). These updates often include security fixes along with non-security fixes. If you are using an older version, your app will be vulnerable."

We may or may not release every patch Tuesday. It is also possible that we ship a security release on a non-patch Tuesday date depending on the severity of the CVE